build-support/bin/json_schema_testdata/BUILD

```diff
@@ -0,0 +1,6 @@
+# Copyright 2023 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+resources(
```

Comment:
I find it helpful to keep the test data outside of the Python file, but if that's deemed unnecessary, happy to move to the Python test module as code instead.